sources/packages/backend/apps/api/src/services/education-program/education-program.service.models.ts

```diff
@@ -35,6 +37,7 @@ export interface SaveEducationProgram {
   isAviationProgram?: string;
   minHoursWeekAvi?: string;
   entranceRequirements: EntranceRequirements;
+  noneOfTheAboveEntranceRequirements: boolean;
```

Comment:
`noneOfTheAboveEntranceRequirements` could be added to EntranceRequirements object right?

Reply:
Yes, sorry. You're right. It was from a previous thing I did and forgot to remove.